variable_selection/fri_kulprit.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
<p>Typo in "coe<strong>e</strong>ficients".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Grupo-de-modelado-probabilista/BART/pull/85/discussion/#comment-1668571923'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>